hello_world.py

```diff
@@ -0,0 +1,13 @@
+#import the Flask class
+from flask import Flask, render_template
+
+#create an instance of Flask class
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+@app.route('/user/<name>')
+def user(name):
+    return render_template('user.html', name=name)
```

Comment:
Add this to end of the file. Notice that we are setting debug=true to print out possible errors on the console:
```suggestion
    return render_template('user.html', name=name)
    
if __name__ == '__main__':
    app.run(debug=True)
```

Reply:
Also, it's completely fine to not commit the `.gitignore` file